scripts/redeploy.py

```diff
@@ -185,10 +185,13 @@ def deploy_tarball(tar_gz_url, build_dir):
             to the /vector directory INSIDE the tarball."
         )
     )
+
+    # --config config.json=../../config.json --config config.localhost.json=./localhost.json
     parser.add_argument(
-        "--config", dest="config", help=(
-            "Write a symlink to config.json in the extracted tarball. \
-            To this location."
+        "--config", action="append", dest="configs",
+        type=lambda kv: kv.split("="), help=(
```

Comment:
I'm a bit concerned about how this will behave when you give it an argument with zero or two =s. Cryptically, I suspect.

could you make it give a sensible error if there is no =, and make it do `kv.split("=", 1)` ?